src/modelsV2/UserAccount.ts

```diff
@@ -0,0 +1,59 @@
+import db from "../db/db";
+import User, { UserToInsert } from "../domain/User";
+
+class UserAccount {
+  public static table = "user_account";
+
+  public static async getAllUsers() {
+    const users = await db(UserAccount.table).select();
+
+    return users;
+  }
+
+  public static async createUser(user: UserToInsert) {
+    const newUser = await db(UserAccount.table).insert(user, [
+      "id",
+      "first_name",
+      "last_name",
+      "birth_date",
+      "ethnicity",
+      "gender",
+      "email",
+    ]);
+
+    return newUser;
+  }
+
+  public static async getUser(userId: number): Promise<User> {
```

Comment:
@mahimag  
would the name getUserById be a better name as we are only comparing with userId?